stdlib/http/src/main/ballerina/src/http/http_client.bal

```diff
@@ -123,57 +123,57 @@ public type HttpClient client object {
     #
     # + path - Request path
     # + request - An HTTP inbound request message
-    # + return - The response for the request or an `ClientError` if failed to establish communication with the upstream server
+    # + return - The response for the request or an `http:ClientError` if failed to establish communication with the upstream server
     public remote function forward(@untainted string path, Request request) returns Response|ClientError {
         return externForward(self, java:fromString(path), request);
     }
 
     # Submits an HTTP request to a service with the specified HTTP verb.
-    # The `HttpClient.submit()` function does not give out a `Response` as the result,
-    # rather it returns an `HttpFuture` which can be used to do further interactions with the endpoint.
+    # The `HttpClient.submit()` function does not give out an `http:Response` as the result.
```

Comment:
Since this is a remote function, we cannot use dot operation right? Someone must have mistakenly added the dot operation. Shall we fix it too?
We can remove HttpClient.submit() or replace it with `HttpClient->submit()`. Need to fix the above methods as well